fasm/output.py

```diff
@@ -0,0 +1,312 @@
+import enum
+from fasm import SetFasmFeature, FasmLine, ValueFormat
+
+def is_only_comment(line):
+    """ Returns True if line is only a comment. """
+    return not line.set_feature and not line.annotations and line.comment
+
+def is_only_annotation(line):
+    """ Returns True if line is only an annotations. """
+    return not line.set_feature and line.annotations and not line.comment
+
+def is_blank_line(line):
+    """ Returns True if line is blank. """
+    return not line.set_feature and not line.annotations and not line.comment
+
+def merge_features(features):
+    """ Combines features with varying addresses but same feature into one feature.
```

Comment:
These examples could be better as doctests?

Reply:
Doctests are the wrong answer given how much is printed.  pytests would be good, but not in this PR.

Reply:
#9